microservices/iptracker-microservice/pom.xml

```diff
@@ -66,6 +66,11 @@
 
   <build>
     <plugins>
+      <plugin>
```

Comment:
declaration without any configuration is most probably redundant

Reply:
default version was complaining sth about JDK 13 here, that's why I updates the plugin version